### ISSUE TITLE
App language in settings doesn't have any effect

### ISSUE THREAD

**Issue by sivaraam** [mtotschnig/MyExpenses#678]
Changing the app languages in the settings to one of the languages other than English does not have any effect. The app languages remains in English even though the setting summary shows that another language is chosen.

I observe this using My expenses 3.1.3.1 on a Samsung SM-J111F running Android 5.1.1 with a device language of English.

**Comment by mtotschnig** [comment]
Thank you for the report. Unfortunately caused by update to Appcompat 1.2.0

**Comment by sivaraam** [comment]
I observe this issue again in 3.1.6.2.

**Comment by mtotschnig** [comment]
Thanks again. Unfortunately the interaction between Appcompat and the language switch feature is very hard to manage. I also observe that the configured language seems to be forgotten on app start, but if you rotate the device, it gets picked up. @sivaraam Could you confirm, if this also works for you?

**Comment by mtotschnig** [comment]
I hope that eb2a5df68194daaf671c12d37909a8e292ff5a71 solves the problem, to be released with 3.1.7 shortly.

**Comment by sivaraam** [comment]
> I also observe that the configured language seems to be forgotten on app start, but if you rotate the device, it gets picked up. @sivaraam Could you confirm, if this also works for you?

Yeah, you're right. I could observe the configured language being used when rotating the device. Really strange.
